e2e.mk

```diff
@@ -0,0 +1,18 @@
+KINDCLUSTER=kind
+
+build-local-readiness: manifests package-vendir-sync
+	COMPONENTS=readiness/controller.readiness-controller-manager.readiness make docker-build-all
+	docker tag projects.registry.vmware.com/tanzu_framework/readiness-controller-manager:latest readiness-controller-manager:latest 
+
+create-kind:
+	kind create cluster --name ${KINDCLUSTER}
+
+deploy-local-readiness: build-local-readiness create-kind
```

Comment:
can we move `create-kind` target to another target `e2e-setup`, since we only need to do this once.

Reply:
Done